plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -222,7 +222,18 @@ func (p *podTolerationsPlugin) getNamespaceTolerationsWhitelist(ns *api.Namespac
 
 func extractNSTolerations(ns *api.Namespace, key string) ([]api.Toleration, error) {
```

Comment:
I take it this method is the one that needs to distinguish between empty and unset?  You definitely need comments.  golang confuses these concepts all over